project/tests/bootstrap.php.twig

```diff
@@ -0,0 +1,39 @@
+<?php
+
+{% if '5.6' not in php %}
```

Comment:
IIRC this should work out of the box of dev-kit

Reply:
do we still need this, if we don't support in anymore?

Reply:
Still supported ATM: https://github.com/sonata-project/dev-kit/blob/6492a2beef15d711b32e3849ddae6668030879dd/config/projects.yml#L11